StackReviewSuggestedEditsRework.user.js

```diff
@@ -669,6 +669,42 @@
         }
     }
 
+    // -------    createUserCard    --------------------
+    // official Stacks documentation: https://stackoverflow.design/product/components/user-cards/
+    function createUserCard(isUserOwner, actionText, actionISO, profileUrl, profileImage, username, reputation, badges) {
```

Comment:
maybe switch the public contract to accept a single configuration object? It's too many positional parameters to not do that, case in point:

```javascript
createUserCard(false, proposedText, proposedISO, null, null, null, null, {});
```